build-constraints.yaml

```diff
@@ -6678,6 +6678,187 @@ packages:
         - zydiskell < 0 # tried zydiskell-0.2.0.0, but its *library* does not support: base-4.15.0.0
         - zydiskell < 0 # tried zydiskell-0.2.0.0, but its *library* does not support: storable-record-0.0.6
 
+        # aeson 2.0
+        - Agda < 0 # tried Agda-2.6.2, but its *library* does not support: aeson-2.0.2.0
```

Comment:
Agda-2.6.2.1 will fix the build problems.  An RC is already out. Release is expected in 2021 still.

Reply:
Agda 2.6.2.1 has been released!